### ISSUE TITLE
The list of ta-lib indicators used in the startegies

### ISSUE THREAD

**Issue by hroff-1902** [freqtrade/freqtrade-strategies#54]
This will help select a replacement library to move to from ta-lib.

`$ egrep -roh 'ta\..*\(' .|sort -u`
gives 
```
ta.ADX(
ta.CCI(
ta.CDLHAMMER(
ta.CMO(
ta.EMA(
ta.MACD(
ta.MAX(
ta.MFI(
ta.MIN(
ta.MINUS_DI(
ta.MOM(
ta.PLUS_DI(
ta.RSI(
ta.SAR(
ta.SMA(
ta.STOCH(
ta.STOCHF(
ta.TEMA(
```

**Comment by hroff-1902** [comment]
Details on apperarances:
```
$ egrep -roh 'ta\..*\(' .|sort|uniq -c
     14 ta.ADX(
     12 ta.CCI(
      1 ta.CDLHAMMER(
      1 ta.CMO(
     45 ta.EMA(
     14 ta.MACD(
      2 ta.MAX(
      8 ta.MFI(
      2 ta.MIN(
      3 ta.MINUS_DI(
      1 ta.MOM(
      2 ta.PLUS_DI(
     17 ta.RSI(
      4 ta.SAR(
     26 ta.SMA(
      1 ta.STOCH(
      8 ta.STOCHF(
      2 ta.TEMA(
```

**Comment by hroff-1902** [comment]
For Finta, only `ta.CDLHAMMER` does not have a replacement.

**Comment by xmatthias** [comment]
I don't think we need to replace talib from the strategies repository to satisfy https://github.com/freqtrade/freqtrade/issues/2509.
talib is a good library - which i think is perfectly fine to include in strategies IF needed (but we should not hard-depend on it as we do now).

We need to remove it from Default / Sample Strategy - and from here "where possible" - maybe.

**Comment by hroff-1902** [comment]
Than it should be added as dependency here. The custom version with the patch...

**Comment by xmatthias** [comment]
> The custom version with the patch...

I'm not sure i understand what you'd patch ...

**Comment by hroff-1902** [comment]
The patch to ta-lib, which is required for cryptos due to low asset price values,
this one:
https://github.com/freqtrade/freqtrade/blob/5e09913e3dc37d2d330be765a48f2722dcc68253/build_helpers/install_ta-lib.sh#L10

**Comment by xmatthias** [comment]
ah that patch ... - sorry - it wasn't really clear.

we'll also need to keep it in freqtrade (imho) itself - removing ta-lib from the docker image will risk breaking peoples running strategies ... which is ... bad . . .

**Comment by hroff-1902** [comment]
it's easier to keep it in the docker (because it's build in a more predeterminated environment and conditions) than to keep it a general dependency...

**Comment by bgi123** [comment]
I think TA modules need to be included in docker images. Some of the strats won't run without them.

**Comment by xmatthias** [comment]
The main TA modules (ta-lib, technical) are included in the docker images.

There's literally 100ds of others out there - if you need one of these, you can use the [docker dependency](https://www.freqtrade.io/en/latest/docker_quickstart/#additional-dependencies-with-docker-compose) instruction to include them in your docker image.
